# message of a single tof sensor
bool is_robot
float32 range